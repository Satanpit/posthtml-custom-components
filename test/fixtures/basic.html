<!doctype html>

<component name="box" mod="mod">

</component>
