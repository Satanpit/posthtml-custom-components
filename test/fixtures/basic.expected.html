<!doctype html>

<div class="box box_mod">

</div>
